src/test/java/org/minions/devfund/TaskOneTest.java

```diff
@@ -0,0 +1,24 @@
+package org.minions.devfund;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+public class TaskOneTest {
+
+    /**
+     * Verify some TaskOne method.
+     */
+    @Test
+    public void testOne() {
+        TaskOne taskOne = new TaskOne();
```

Comment:
You are repeating this line of code in all your tests.
Please move it to a Before method.

Reply:
done